app/Jobs/ApplicationDeploymentJob.php

```
@@ -2125,16 +2242,74 @@ private function pull_latest_image($image)
+            ],
+            [
+                executeInDocker($this->deployment_uuid, 'cat /artifacts/build.sh'),
+                'hidden' => true,
+            ],
+            [
+                executeInDocker($this->deployment_uuid, 'bash /artifacts/build.sh'),
+                'hidden' => true,
+            ]
+        );
+        $this->application_deployment_queue->addLogEntry('Building docker image completed.');
+    }
```

PHPMD (2.15.0)

The method build_static_image is not named in camelCase. (undefined)

(CamelCaseMethodName)